lib/lists/list_entitylistview.dart

```diff
@@ -21,6 +23,13 @@ class EntityListView extends ConsumerStatefulWidget {
   ConsumerState<EntityListView> createState() => _EntityListViewState();
 }
 
+List<MapEntry<int, QueryDocumentSnapshot<Map<String, dynamic>>>> sortList(
+    List<MapEntry<int, QueryDocumentSnapshot<Map<String, dynamic>>>> list1) {
+  return list1
+    ..sort(
+        (e1, e2) => e1.value.data()['name'].compareTo(e2.value.data()['name']));
```

Comment:
This should not rely on 'name' but instead rely on the entityName1 or data()['entityName1']+' '+data()['entityName2'] values...

![image](https://user-images.githubusercontent.com/2824906/217391921-aadce382-043a-48b8-9d9a-81b0d3893ed5.png)

Reply:
Using entityname1 to sort as when using entityname1 and entityname2 , getting error in UI
```
 destinationList = sourceList
        ..sort((e1, e2) =>
            (e1.value.data()['entityName1'] + e1.value.data()['entityName2'])
                .compareTo(e2.value.data()['entityName1'] +
                    e2.value.data()['entityName2']));
```
![image](https://user-images.githubusercontent.com/105192215/218307704-11edbba7-e425-44e4-9d04-5ae3df65604a.png)

Reply:
Also, entityname1 and entityname2 fields are not present in the map we are sorting here, we either have name or entity number,
![image](https://user-images.githubusercontent.com/105192215/218308028-2e4a8ed1-5c17-47ae-b14e-481fefa53515.png)